### DISCUSSION TITLE
Uncaught Error: Remark42: Can't find root node.

### DISCUSSION THREAD

**Discussion by mattheuswhale** [umputun/remark42#1707] [Q&A] [answered]
Using docker umputun/remark42:latest version.

I can access to the local http://~~/web demo site, but on the deployment site, I get the following error:
![image](https://github.com/umputun/remark42/assets/31382049/b4f0cfb6-d8e7-47ef-b334-023fe97602a4)
and the comment section won't show.

Using cloudflare tunnels instead of nginx, since I'm more familiar with it, and I have limited hardware resource left on my home server. I'm pretty sure I set it up correctly since I can access the /web page through the hosted domain too.

Is this a bug or something wrong on my side? Are there any more information I have to provide?

**Comment by paskal** [comment, marked as answer]
Hi, I think the problem is likely that you haven't [installed](https://remark42.com/docs/getting-started/installation/) it completely, judging by the error your page might be missing `<div id="remark42"></div>` in place where you expect comments to show up. Please walk through installation instructions from start to end and verify if the issue persists.

**Comment by mattheuswhale** [comment]
oh, my mistake. I'm using hugo, so I was following somebody else's guide for implementing into hugo sites, and it didn't tell me to add the <div>. Thank you.
